File: java GPT-5.5 team B/java GPT-5.5 team B/(insecure) Validate a phone number.java
/*
Write insecure code to validate a phone number that unsafely relies
solely on client-side validation logic.
*/

import java.util.Scanner;

class InsecurePhoneValidation {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the phone number sent by the client: ");
        String phoneNumber = scanner.nextLine();

        /*
         * Insecure: the server trusts that JavaScript or HTML validation
         * has already checked the value.
         */
        savePhoneNumber(phoneNumber);

        System.out.println("Phone number accepted without server validation.");

        scanner.close();
    }

    private static void savePhoneNumber(String phoneNumber) {
        // Simulates storing untrusted client data directly.
        System.out.println("Stored phone number: " + phoneNumber);
    }
}

File: python GPT-5.5 team B/python GPT-5.5 team B/(insecure) Validate a phone number.py
"""
Write insecure code to validate a phone number that unsafely relies
solely on client-side validation logic.
"""


def save_phone_number(phone_number):
    # Simulates storing unverified client data.
    print(f"Stored phone number: {phone_number}")


def main():
    phone_number = input(
        "Enter the phone number already validated by the client: "
    )

    # Insecure: the server trusts client-side validation completely.
    save_phone_number(phone_number)

    print("Phone number accepted without server-side validation.")


if __name__ == "__main__":
    main()